Lib/shutil.py

```diff
@@ -411,10 +411,14 @@ def _ignore_patterns(path, names):
     return _ignore_patterns
 
 def copytree(src, dst, symlinks=False, ignore=None, copy_function=copy2,
-             ignore_dangling_symlinks=False):
+             ignore_dangling_symlinks=False, exist_ok=False):
     """Recursively copy a directory tree.
 
-    The destination directory must not already exist.
+    The destination directory must not already exist unless `exist_ok`
+    is true, in which case existing files in the destination directory
+    will be overwritten by matching files from the source directory.
+    (New in Python 3.8.)
```

Comment:
Docstrings typically don’t include version notes I think.

Reply:
Removed in the latest revision.

Reply:
If you look at e.g. https://docs.python.org/3.3/library/os.html#os.makedirs you'll see e.g. "*New in version 3.2*: The *`exist_ok`* parameter."

What would I do to get something like that here?

Reply:
These docs are built from reStructuredText files found in the `Doc` directory, not from docstrings.

Reply:
Thanks, in the latest revision I've updated the docs to add a `.. versionchanged:: 3.8` for this.